cpp/src/gandiva/tree_expr_test.cc

```diff
@@ -45,7 +45,7 @@ class TestExprTree : public ::testing::Test {
   FieldPtr i1_;  // int32
 
   FieldPtr b0_;  // bool
-  FunctionRegistry registry_;
+  FunctionRegistry& registry_ = *gandiva::default_function_registry();
```

Comment:
Can we use `std::shared_ptr<FunctionRegistry>` here too?

Reply:
Sure. Updated.